### DISCUSSION TITLE
Could I get signature from useConnect?

### DISCUSSION THREAD

**Discussion by yenche123** [unicape/use-wagmi#87] [Q&A] [answered]
I would like to check out if the address belongs to the current user, so is it possible to get the signature from `useConnect()` and let me pass it to my backend?
If so, I can verify the address on backend to allow user  to sign in.
Last but not last, I am really appreciated to your work on this project. Thanks a lot!

**Comment by unicape** [comment, marked as answer]
@yenche123 
https://wagmi.sh/react/hooks/useSignMessage
https://wagmi.sh/react/hooks/useSignTypedData

**Reply by yenche123** [reply]
ok! Thanks!!
